### DISCUSSION TITLE
seeing the note's whole body

### DISCUSSION THREAD

**Discussion by gour** [ahyatt/ekg#116] [Q&A] [answered]
Hello!

I have a need to sometimes capture somewhat longer notes, e.g.: 130 lines, 20 sentences, 829 words, and 5041 characters. However, when I do `ekg-show-notes-with-any-tag`, the buffer does display only part of it: 67 lines, 13 sentences, 518 words, and 3099 characters, so wonder if there is some setting to allow showing the full note's text body?

**Comment by ahyatt** [comment, marked as answer]
Yes, `ekg-display-note-template`.  The default is: `%n(id)%n(tagged)%n(titled)%n(text 500)%n(other)`, so it will show 500 words of text from the input.  You can change that to any other value.  The manual has [more information](https://github.com/ahyatt/ekg/blob/main/doc/ekg.org#73-inline-commands). 

I'm going to close this comment, but feel free to open again if my answer isn't sufficient.
